releases/unreleased/remove-groups-table-from-ui.yml

```
@@ -0,0 +1,10 @@
+---
+title: Remove groups table from UI
```

```suggestion
title: Groups table removed from the UI
```

The title should be reworded because it's for the end user and not the title of the commit.